make/mt-os/mt-stm32f4_nucleo_wifi.mk

```diff
@@ -16,6 +16,8 @@ XI_COMPILER_FLAGS += -I$(XI_STM32_PATH_SDK)/Drivers/CMSIS/Device/ST/STM32F4xx/In
 XI_COMPILER_FLAGS += -I$(XI_STM32_PATH_SDK)/Drivers/STM32F4xx_HAL_Driver/Inc
 XI_COMPILER_FLAGS += -I$(XI_STM32_PATH_SDK)/Drivers/BSP/STM32F4xx-Nucleo
 
+XI_COMPILER_FLAGS += -I$(LIBXIVELY)/examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Inc
```

Comment:
last spring Kotyo worked on cut the dependency between Xively C Client and its examples. Reason is that a nucleo wifi libxively build can have multiple example using it. Thus the libxively cannot depend on their usage places.